care/facility/api/serializers/ambulance.py

```diff
@@ -55,12 +56,12 @@ def create(self, validated_data):
 
     def update(self, instance, validated_data):
         validated_data.pop("drivers", [])
-        ambulance = super(AmbulanceSerializer, self).update(instance, validated_data)
+        ambulance = super().update(instance, validated_data)
         return ambulance
 
 
 class DeleteDriverSerializer(serializers.Serializer):
-    driver_id = serializers.IntegerField()
+    driver_id = serializers.UUIDField()
```

Comment:
this endpoint is not being used on the frontend so it is okay to change to ext id